apps/playground/src/app/modules/components/tag-module/tag-module.component.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Angular imports
+ */
+import { Component } from '@angular/core';
+import { NonNullableFormBuilder } from '@angular/forms';
+
+/**
+ * 3rd-party imports
+ */
+import { ElementSize } from '@betagouv/ngx-dsfr';
+import { TagType } from '../../../../../../../libs/ngx-dsfr/tag/src/lib/tag.component';
```

Comment:
After you've changed the imports in the `index` file of the `DsfrTagComponent`, you'll be able to change here to this : `import { TagType } from '@betagouv/ngx-dsfr/tag';`